### PR TITLE
Copyediting `README.md`

### PR DESCRIPTION
**Pull request by OCRenkist**
- Some modest improvements to the visual appearance.
- Added hyperlink display text.
- More monospace formatting for technical terms.

### CODE REVIEW

**Review by appgurueu — APPROVED** [review]
The padding of `name` and `author` isn't really needed IMO.

File: game.conf [hunk 1]
```diff
@@ -1,3 +1,3 @@
-name = Minetest Game
-author = Minetest
-description = A basic exploration, mining, crafting and building sandbox game with no NPCs, monsters or animals. Minetest Game is usually used with mods added and many mods are available for this game. Reliably maintained by Minetest Engine core developers.
+name        = Minetest Game
+author      = Minetest
+description = A basic exploration, mining, crafting, and building, sandbox game with no NPCs, monsters, or animals. Minetest Game is usually used with mods added, and many mods are available for this game. Reliably maintained by Minetest Engine core developers.
```
Comment (sfan5) [inline game.conf:3]: A comma before `and` or `or`? I'm not a native speaker but that doesn't look right to me.

Comment (appgurueu) [reply game.conf:3]: This is called to as ["Serial comma"](https://en.wikipedia.org/wiki/Serial_comma). Both are correct in English.

Comment (OCRenkist) [reply game.conf:3]: "foo, bar and baz" = `(foo and (bar and baz))`
"foo, bar, and baz" = `(foo and bar and baz)`
_Aber ich bin nur deutschsprechend Ausländer._ &nbsp; b^_~

**Review by sfan5 — APPROVED** [review]

**Pull request merged**